packages/mds-compliance/mds-compliance-engine.ts

```diff
@@ -353,8 +342,8 @@ function processPolicy(
     return {
       policy,
       compliance,
-      total_violations: overflowedVehicles.length + allSpeedingVehicles.length,
-      vehicles_in_violation: { ...overflowedVehicles, ...speedingVehiclesMap }
+      total_violations: overflowedVehicles.length + allTimeVehicles.length + allSpeedingVehicles.length,
+      vehicles_in_violation: { ...overflowedVehicles, ...timeVehiclesMap, ...speedingVehiclesMap }
```

Comment:
we need better testing/better type annotations. i just realized the type of `timeVehiclesMap` and `speedingVehiclesMap` do not match `overflowedVehicles`, but tests passed regardless.